@thomas-fossati editorial pass: manageability (#74)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

Handshake RTT can be measured by adding the client-to-observer and
observer-to-server RTT components together. This measurement necessarily
includes any transport and application layer delay (the latter mainly

> ascribable to the asymmetric crypto operations associated with the TLS

Author: "ascribable" is possibly confusing to non-native speakers; suggest simplification.

```suggestion
caused by the asymmetric crypto operations associated with the TLS
```

# Specific Network Management Tasks

In this section, we review specific network management and measurement
techniques and how QUIC's design impacts them.

## Stateful treatment of QUIC traffic {#sec-stateful}

> Stateful treatment of QUIC traffic (e.g., at a firewall or NAT middlebox) is possible through QUIC traffic and version

Author: rewrap this para (and possibly a few others); the 80col linter will choke on it.
